src/components/AirbaseNG/AirbaseNG.tsx

```diff
@@ -1,165 +1,171 @@
-    const [output, setOutput] = useState(""); // State variable to store the output of the command execution.
-    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
-    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    // State variables to manage the component's functionality.
+    const [loading, setLoading] = useState(false);
+    const [output, setOutput] = useState("");
+    const [pid, setPid] = useState("");
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false);
     const [allowSave, setAllowSave] = useState(false);
     const [hasSaved, setHasSaved] = useState(false);
-    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
-    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
+    const [opened, setOpened] = useState(!isCommandAvailable);
+    const [loadingModal, setLoadingModal] = useState(true);
     const [advanceMode, setAdvanceMode] = useState(false);
     const [customMode, setCustomMode] = useState(false);
 
-    // Component Constants.
-    const title = "Airbase-ng"; // Title of the component.
-    const description = "Airbase-ng is a tool to create fake access points."; // Description of the component.
+    // Constants for the component's metadata.
+    const title = "Airbase-ng";
+    const description = "Airbase-ng is a tool to create fake access points.";
     const steps =
         "Step 1: Type in the name of your fake host.\n" +
         "Step 2: Select your desired channel.\n" +
         "Step 3: Specify the WLAN interface to be used.\n" +
         "Step 4: Click 'Start AP' to begin the process.\n" +
-        "Step 5: View the output block to see the results. ";
-    const sourceLink = ""; // Link to the source code (or Kali Tools).
-    const tutorial = ""; // Link to the official documentation/tutorial.
-    const dependencies = ["aircrack-ng"]; // Contains the dependencies required by the component.
+        "Step 5: View the output block to see the results.";
+    const sourceLink = ""; // Placeholder for source link.
+    const tutorial = ""; // Placeholder for tutorial link.
+    const dependencies = ["aircrack-ng"];
 
-    // Form hook to handle form input.
+    // Form handling for the component's input fields.
     const form = useForm({
         initialValues: {
             fakeHost: "",
-            channel: "",
-            replayInterface: "",
+            channel: "6", // Default channel
+            replayInterface: "wlan0", // Default interface
             macAddress: "",
             filePath: "",
+            verbosity: false,
+            disableBeacons: false,
             customConfig: "",
+            customScript: "",
+        },
+
+        // Validation: Ensure essential fields are filled.
```

Comment:
Is this validation required??? It looks like Mantine already has validation for required fields: 

![image](https://github.com/user-attachments/assets/d4549469-3bd7-4ba9-af70-c6357ea8d499)